### PR TITLE
Update AuthorNotification.md

### PR DESCRIPTION
**Pull request by v-caxian**

### CODE REVIEW

**Comment by PRMerger10** [comment]
@v-caxian : Thanks for your contribution! The author, @PRMerger16, has been notified to review your proposed change.

File: csidevdocs-public/Demo/v-caxian/AuthorNotification.md [hunk 1]
```diff
@@ -46,4 +46,4 @@ Sogup
 To effectively use the OPS Authoring extension, you must1:
```
Comment (PRMerger16) [inline csidevdocs-public/Demo/v-caxian/AuthorNotification.md]:
```suggestion
To effectively use the OPS Authoring extension, you must2:
```

File: csidevdocs-public/Demo/v-caxian/AuthorNotification.md [hunk 2]
```diff
@@ -46,4 +46,4 @@ Sogup
...
 - Clone your entire repo to your local machine and keep it in sync. Functions such as link and image insertion are not reliable if the repo is out of sync.
 - For accurate content preview, you need to update your VS Code settings.json file, as described under **Preview Content** below.
```
Comment (PRMerger16) [inline csidevdocs-public/Demo/v-caxian/AuthorNotification.md]:
```suggestion
- For accurate content preview, you need to update your VS Code settings file, as described under **Preview Content** below.
```

**Comment by PRMerger10** [comment]
Microsoft authors of articles need to make updates through the private repository so the updates can be staged and validated by the current validation rules. After you review staging and fix all validation issues, you can add the sign-off comment to let the PR reviewers know the updates are ready to be merged.It's easy to access the private repo version of your articles in GitHub. Just add -pr to the URL, as follows: 
 Public repo URL: https://github.com/Microsoft/azure-docs/blob/master/articles/azure-portal-overview.md 
 Private repo URL: https://github.com/Microsoft/azure-docs-pr/blob/master/articles/azure-portal-overview.md 
 We have closed this pull request. Please resubmit the updates through the private repo. If you need perms to the private repo, [see the contributor guide instructions](https://github.com/Microsoft/azure-docs/blob/master/contributor-guide/tools-and-setup.md#permissions)

**Pull request closed without merging**